multiversx_sdk_core/transaction_factories/transaction_builder.py

```diff
@@ -0,0 +1,57 @@
+from typing import List, Optional, Protocol
+
+from multiversx_sdk_core.constants import ARGS_SEPARATOR
+from multiversx_sdk_core.interfaces import IAddress
+from multiversx_sdk_core.transaction import Transaction
+
+
+class IConfig(Protocol):
+    chain_id: str
+    min_gas_limit: int
+    gas_limit_per_byte: int
+
+
+class TransactionBuilder:
```

Comment:
Until `transaction_builders` gets removed (v2), we'll have 2 `TransactionBuilder` objects. Hopefully, it will be all right.